apps/prairielearn/src/pages/instructorAssessmentManualGrading/assessmentQuestion/assessmentQuestion.ts

```diff
@@ -136,6 +139,22 @@ router.post(
       } else {
         res.send({});
       }
+    } else if (req.body.__action === 'bot_grade_assessment') {
+      // check if bot grading is enabled
+      const bot_grading_enabled = await features.enabledFromLocals('bot-grading', res.locals);
+      if (!bot_grading_enabled) {
+        throw new error.HttpStatusError(403, 'Access denied (feature not available)');
+      }
+
+      const jobSequenceId = await botGrade({
+        question: res.locals.question,
+        course: res.locals.course,
+        course_instance_id: res.locals.course_instance.id,
+        assessment_question: res.locals.assessment_question,
+        urlPrefix: res.locals.urlPrefix,
+        authn_user_id: res.locals.authz_data.user.user_id,
```

Comment:
```suggestion
        authn_user_id: res.locals.authn_user.user_id,
```

You should also pass `res.locals.user.user_id` through to `userId` in `createServerJob`:

https://github.com/PrairieLearn/PrairieLearn/blob/2c3624e7201e2b73412ba0e8fe0af000ab421b7a/apps/prairielearn/src/lib/server-jobs.ts#L291

This is all important to make sure we have proper audit trails in place for everything.